pkg/apis/keycloak/v1alpha1/keycloakclient_types.go

```diff
@@ -93,7 +98,7 @@ type KeycloakAPIClient struct {
 	Attributes map[string]string `json:"attributes,omitempty"`
 	// True if Full Scope is allowed.
 	// +optional
-	FullScopeAllowed bool `json:"fullScopeAllowed,omitempty"`
+	FullScopeAllowed *bool `json:"fullScopeAllowed,omitempty"`
```

Comment:
This change should probably be excluded from this PR due to https://github.com/keycloak/keycloak-operator/pull/282

Reply:
The commit is merely included in the PR to make the PR testable on it's own. As written in the PR description I'll rebase when that commit is in master.